affirm_example/templates/index.html

```diff
@@ -144,6 +154,12 @@ <h3>Complete your checkout</h3>
     var currency_label = document.getElementById('currency');
     currency_label.innerText = currency;
   }
+
+  function updateMerchant() {
+    var merchant_public_api_key = document.getElementById('merchant').value;
+    checkoutData.merchant.public_api_key = merchant_public_api_key
+    checkoutData.merchant.user_confirmation_url = checkoutData.merchant.user_confirmation_url.replace(/\?public_api_key=.+/, '?public_api_key=' + merchant_public_api_key)
```

Comment:
nit: do you want to use `URL` here so it's less likely this could go wrong?

like 
```
var url = new URL(checkoutData.merchant.user_confirmation_url)
url.searchParams.set('public_api_key', merchant_public_api_key)
checkoutData.merchant.user_confirmation_url = url.toString();
```

but if we need to support IE for example, might need to include polyfill as well... but maybe we don't need to support IE?

Reply:
[0349d73](https://github.com/Affirm/affirm-example-python/pull/32/commits/0349d73810e41c74327fabb4d3c9874ebfb989f7)